services/web-server/src/login/strategies/mozilla-auth0.js

```diff
@@ -28,11 +28,7 @@ export default class MozillaAuth0 {
 
   // Get a personAPI instance, by requesting an API token as needed.
   // See https://github.com/mozilla-iam/cis/blob/f90ba5033785fd4fb14faf9f066e17356babb5aa/docs/PersonAPI.md#do-you-have-code-examples
-  async getPersonApi() {
-    if (this._personApi && new Date().getTime() / 1000 < this._personApiExp - 10) {
```

Comment:
the real issue was here.
`_personApiExp` was already in ms.. so there was no need for an extra `/1000`